scripts/prepare_release.sh

```diff
@@ -28,16 +28,14 @@ git reset --hard origin/main
 git checkout -b release/${VERSION}
 git push origin release/${VERSION}
 
-# create a temporary branch to create a PR for updated version and changelogs
-git checkout -b release/${VERSION}-auto
 ./scripts/eachdist.py release --version ${VERSION}
 rc=$?
 if [ $rc != 0 ]; then
     echo "::set-output name=version_updated::0"
     exit 0
 fi
 
-git add **/version.py **/setup.cfg **/CHANGELOG.md
+git add .
```

Comment:
Should we keep this to be safe incase we modify eachdist.py without changing this?